open-api/overview.md

```diff
@@ -0,0 +1,10 @@
+---
+slug: /overview
+id: open-api-overview
+sidebar_position: 0
+---
+
+# Open API Overview
+
+
+## This is an overview of the open api
```

Comment:
Add some introduction here. For example:

* What is the sCrypt Open API?
* How to get a API Key?
* ....